aws-msk/config.yml

```diff
@@ -0,0 +1,41 @@
+# Name of the quickstart (required)
+name: template-quickstart
+
+# Displayed in the UI (required)
+title: Template Quickstart
+
+# Long-form description of the quickstart (required)
+description: |
+  The template quickstart allows you to get visibility into the performance and available of your example service and dependencies. Use this quickstart together with the mock up integrations.
+
+# Displayed in search results and recommendations. Summarizes a quickstarts functionality.
+summary: |
+  A short form description for this quickstart
+
+# Support level: New Relic | Verified | Community (required)
+level: Community
+
+# Authors of the quickstart (required)
+authors:
+  - John Smith
+
+# Keywords for filtering / searching criteria in the UI
+keywords:
+  - list
+  - of
+  - searchable
+  - keywords
+
+# Reference to install plans located under /install directory
+# Allows us to construct reusable "install plans" and just use their ID in the quickstart config
+installPlans:
+  - infra-agent-standard
+
+documentation:
+  - name: Installation Docs
+    url: docs.newrelic.com
+    description: Description about this doc reference
+
+# Content / Design
+logo: logo.png
```

Comment:
Is the image at `images/aws-msk-logo.svg` what you want to use as the logo for this quickstart? If so, you'll just need to update this path to `/image/aws-msk-logo.svg`